### PR TITLE
Added recipe for Flask-BasicAuth

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

**Comment by pmlandwehr** [comment]
Looks like circle CI crashed due to some binstar/anaconda.org issue; close/reopening

**Comment by pmlandwehr** [comment]
Ok, not sure what's up here. Thoughts, @jakirkham?

**Comment by sodre** [comment]
@jakirkham can we force the rebuild of this package by closing and reopening the PR? 
I looked through the build-logs and it looks like the Continuum was having problems during the build time.

**Comment by pmlandwehr** [comment]
Giving the old close/reopen another try

**Comment by sodre** [comment]
The only other thing I noticed is that it pulling flask from `defaults` instead of from `conda-forge`. I wonder if that is part of the problem.

Sorry I haven't been more helpful.

**Comment by pmlandwehr** [comment]
That's still surprising - I would have expected the repo to prioritize conda-forge over the defaults.

**Comment by sodre** [comment]
the file it failed to download "https://conda.anaconda.org/conda-forge/linux-64/jinja2-2.8-py34_1.tar.bz2" worked just fine from my home. This is puzzling.

**Comment by sodre** [comment]
Also, this last time it failed in circle-ci and not the AppVeyor test, which makes it even more puzzling. Can you give it one more close-reopen to see if goes through all three tests ?

**Comment by jakirkham** [comment]
Restarted and it now passes. No clue what was going on before, but does seem like there were difficulties with Anaconda.org. Let's keep our eyes open for this. Thanks for helping out @sodre.

File: recipes/flask-basicauth/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - flask
```
Comment (jakirkham) [inline recipes/flask-basicauth/meta.yaml]: Probably can drop `flask` from build.

**Comment by jakirkham** [comment]
One comment, but am going to let it be addressed in the feedstock to keep things moving. Thanks both of you for working on this.

**Pull request merged**